apps/expo/package.json

```diff
@@ -15,9 +15,10 @@
     "typecheck": "tsc --noEmit"
   },
   "dependencies": {
+    "@acme/api": "workspace:*",
```

Comment:
Have Expo shipped tree shaking yet? If not I think including this as a prod dep will leak backend code

Reply:
https://x.com/anis_RNCore/status/1841790356930887859

Reply:
Good point, didn't though of that. I'm not sure how to check if it's happening (I'm quite new with Expo and RN). Also, not sure if updating SDK is easy or not and should be done in this PR or in a different one.

Another option would be to move the transformer logic into a separate package so we are 100% sure it doesn't bring any BE code. For example `@acme/transformer` or `@acme/api-transformer`

Reply:
I am personally scared of adding an api package as a dependency, regardles of wether or not Expo does tree-shaking correctly or not. Might not be an issue, but it's something to be mindful of.

I personally have a `@acme/shared` package that exports common utils/stuff  that is safe to be shared everywhere. Adding another package just for the tranformer export might be a bit verbose for this repo though

Reply:
Cool, yeah it makes sense. I'll prepare that and revert the api package to just dev dependency in Expo.